src/app/components/Operator/components/ValidatorsCount.tsx

```diff
@@ -41,15 +39,10 @@ const ValueContent = styled.div`
 `;
 
 const ValidatorCount = ({ validatorCount }: { validatorCount: number }) => {
-    const headerTooltipStyle = { fontSize: '14px', color: 'rgb(161, 172, 190)', marginBottom: '-2px' };
-
     return (
       <ValidatorCountContainer>
         <DataHeader>
           Validators
-          <InfoTooltip
-            style={headerTooltipStyle}
-            message="Is the operator performing duties for the majority of its validators in the last 10 epochs." />
```

Comment:
Should ask @yoanatanblox how and where this information should be visible to end-user

Reply:
@meshin-dev He asked to remove this tooltip you can see it under this ticket in comments https://blox-ensemble.monday.com/boards/4755515647/pulses/4942354386